src/main/java/bowling/domain/dto/FrameResponse.java

```diff
@@ -0,0 +1,22 @@
+package bowling.domain.dto;
+
+import bowling.domain.frame.Frame;
+
+public class FrameResponse {
```

Comment:
해당 클래스는 Frame을 담아서 이동만 하고 있는 걸로 보이는데요. 어떤의도인지 알기 어렵네요.. 💭 
domain을 DTO로 변환시키는것이 목적이라면 필요한 값들을 가지고 있는 구조는 어떤가요?
```suggestion
public class FrameResponse {
    private PitchState pitchState;
    //...
```